tests/DoctrineTest/InstantiatorTestAsset/SerializableArrayObjectAsset.php

```diff
@@ -37,6 +37,7 @@ class SerializableArrayObjectAsset extends ArrayObject implements Serializable
      */
     public function __construct()
     {
+        parent::__construct();
```

Comment:
Wouldn't add these

Reply:
couldn't see a quick way to disable a single rule....
maybe i'm not groking the phpstan well enough

Reply:
hmm, just have to use `ignoreMessages`?

Reply:
yep